internal/pkg/v2/infrastructure/redis/client.go

```diff
@@ -281,6 +281,22 @@ func (c *Client) AddDevice(d model.Device) (model.Device, errors.EdgeX) {
 	return addDevice(conn, d)
 }
 
+// Delete all readings and events
+func (c *Client) DeleteAllEvents() errors.EdgeX {
+	conn := c.Pool.Get()
+	defer conn.Close()
+
+	err := scrubRecordsByKey(conn, ReadingsCollection)
+	if err != nil {
+		return errors.NewCommonEdgeX(errors.KindDatabaseError, "scrub all readings failed", err)
+	}
+	err = scrubRecordsByKey(conn, EventsCollection)
+	if err != nil {
+		return errors.NewCommonEdgeX(errors.KindDatabaseError, "scrub all events failed", err)
```

Comment:
change message to `delete all pushed  events failed`

Reply:
Refactored